Project1/Index.html

```diff
@@ -5,28 +5,31 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Oswald:500" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Libre+Franklin" rel="stylesheet">
     <title>Document</title>
 </head>
 
 <body>
-    <h1>Yes, the Octopus Is Smart as Heck. But Why?
-        It has eight arms, three hearts — and a plan. Scientists aren’t sure how the cephalopods got to be so
-        intelligent.</h1>
-    <h2>By Carl Zimmer
-        Nov. 30, 2018</h2>
+    <header>
+        <figure>
+            <img src="images/maxresdefault.jpg" alt=" Octopus on the sea bed">
+            <figcaption>An image of an octopus via youtube. </figcaption>
```

Comment:
Since you reference YouTube in your caption, maybe link to the original source here?